tensornetwork/tests/network_test.py

```diff
@@ -96,7 +96,8 @@ def test_network_copy_names(backend):
   c = net.add_node(np.random.rand(3, 3, 3), name='c')
   a[0] ^ b[1]
   b[2] ^ c[0]
-  _, node_dict, edge_dict = net.copy()
+  conj=False
```

Comment:
Please do not modify existing tests. Please make a new test and compare the before/after Tensors of a conjugated copy.